test/e2e/helpers.go

```diff
@@ -292,7 +310,7 @@ func execOnHost(controlPlaneEndpoint, hostname string, f io.StringWriter, comman
 	if err != nil {
 		return err
 	}
-	port := "22" // Need to use port 50001 for VMSS when MachinePools are supported here.
+	port := e2eConfig.GetVariable(VMSSHPort) // Or VMSSSSHPort when MachinePools are supported
```

Comment:
in the future, should this be passed in as a parameter?

Reply:
Yes, it obviously should. I can make that change before we merge this.

Reply:
Done.